draft-ietf-ace-extend-dtls-authorize.md

```diff
@@ -62,14 +62,14 @@ entity:
 
 --- abstract
 
-This document updates the CoAP-DTLS profile for ACE by specifying that
-the profile applies to TLS as well as DTLS.
+This document updates the CoAP-DTLS profile for ACE described in RFC 9202
+by specifying that the profile applies to TLS as well as DTLS.
 
 --- middle
 
 # Introduction
 
-{{RFC9202}} only specifies the use of DTLS {{RFC6347}} {{RFC9147}} but works equally well for TLS {{RFC8446}}. For many constrained implementations, CoAP over UDP {{RFC7252}} is the first choice, but when deploying ACE in networks controlled by other entities (such as the Internet), UDP might be blocked on the path between the client and the RS, and the client might have to fall back to CoAP over TCP {{RFC8323}} for NAT or firewall traversal. This feature is supported by the OSCORE profile {{RFC9203}} but is lacking in the DTLS profile.
+{{RFC9202}} only specifies the use of DTLS {{RFC6347}} {{RFC9147}} but works equally well for TLS {{RFC8446}}. For many constrained implementations, CoAP over UDP {{RFC7252}} is the first choice, but when deploying ACE in networks controlled by other entities (such as the Internet), UDP might be blocked on the path between the client and the RS, and the client might have to fall back to CoAP over TCP {{RFC8323}} for NAT or firewall traversal. This dual support for TLS and DTLS is already supported by the OSCORE profile {{RFC9203}}.
```

Comment:
The problem with this change is that "the dual support for TLS and DTLS" ist not supported by the OSCORE profile because OSCORE achieves its goals by being transport protocol agnostic. I was stuck on this for a while to come up with a more accurate description.

Reply:
How about "This dual support for security over TCP as well as UDP is already supported by the OSCORE profile {{RFC9203}}."

Reply:
This is fine by me.